torch/distributed/distributed_c10d.py

```diff
@@ -971,17 +971,18 @@ def _is_barrier_after_init() -> int:
     return int(os.getenv("TORCH_DIST_INIT_BARRIER", "0"))
 
 
-def _get_default_group():
+def _get_default_group() -> ProcessGroup:
     """Get the default process group created by init_process_group."""
     if not is_initialized():
         raise ValueError(
             "Default process group has not been initialized, "
             "please make sure to call init_process_group."
         )
+    assert GroupMember.WORLD is not None
     return GroupMember.WORLD
 
 
-def _get_default_store():
+def _get_default_store() -> Optional[Store]:
```

Comment:
is this actually Optional?

Reply:
good catch, the underlying map is unnecessarily `Optional[Store]`; will fix